activities/understanding-our-users/enabling-policies.md

```diff
@@ -0,0 +1,22 @@
+---
+# SPDX-License-Identifier: CC0-1.0
+# SPDX-FileCopyrightText: 2023 The Foundation for Public Code <info@publiccode.net>
+type: Resource
+---
+
+# Enabling policies
+
+If there are policies supporting open source development it can be easier for civil servants to convince their peers that they should engage.
+These policies can take many forms, from laws to internal policies.
```

Comment:
This lets us avoid a 'policies can take many forms, including policies' recursive argument.
```suggestion
These policies can take many forms, from laws to internal guidance.
```